templates/batchConversion.tpl

```diff
@@ -0,0 +1,52 @@
+{**
+ * templates/batchConversion.tpl
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2003-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * markup plugin -- displays the ArticlesGrid.
+ *}
+<div>
+	<div class="pkp_form">
+		<fieldset>
+			<legend>Selected submissions</legend>
```

Comment:
Move hard-coded text into the locale files

Reply:
I will remove that legend since we no longer select submissions.

Reply:
https://github.com/kaschioudi/ojs3-markup/commit/9c177a2d3697683c734754be13cf2737552358f8